spark/src/main/java/org/apache/iceberg/spark/SparkWriteOptions.java

```diff
@@ -50,4 +50,7 @@ private SparkWriteOptions() {
 
   // File scan task set ID that indicates which files must be replaced
   public static final String REWRITTEN_FILE_SCAN_TASK_SET_ID = "rewritten-file-scan-task-set-id";
+
+  // Partition spec ID to use when writing new files
+  public static final String SPEC_ID = "spec-id";
```

Comment:
should we make the constant from `PartitionSpecParser` public? then we don't have define it here as it is a more general constant than spark

Reply:
I need to actually delete this, I had an implementation for the spark writer using this property but it did not work. I"ll remove this for now